Rakefile

```diff
@@ -408,6 +408,41 @@ We use information from your github account throughout the class.
         end,
       },
 
+      mongodb:{
+        header: "MongoDB",
+        installation_steps: [
+          %q(
+1. Install mongodb with brew
+
+    $ brew install mongodb
+
+2. Create the folder mongo will be using to store your databases
+
```

Comment:
This may already be in installfest, but add a note that `sudo`  will prompt you for your password, but that you will not be able to see output related to your typing.